tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -171,3 +171,30 @@ Feature: collaboration (wopi)
       | app-endpoint                                     |
       | /app/open?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open?file_id=<<FILEID>>                     |
+
+
+  Scenario Outline: open file with .odt extension in web
+    Given user "Alice" has uploaded file "filesForUpload/simple.odt" to "simple.odt"
+    And we save it into "FILEID"
+    When user "Alice" sends HTTP method "POST" to URL "<app-endpoint>"
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "uri"
+        ],
+        "properties": {
+          "uri": {
+            "type": "string",
+             "pattern": "^.*?contextRouteName=files-spaces-personal&fileId=%file_uuidv4_pattern%%24%file_uuidv4_pattern%%21%file_uuidv4_pattern%$"
```

Comment:
what's the actual URL here?

Reply:
If included `app_name` in endpoint then it is:
`https://host.docker.internal:9200/external?app=FakeOffice\u0026contextRouteName=files-spaces-personal\u0026fileId=c3ce162e-6dd4-4abc-b402-d50d385a0573%24d08915f8-9abb-4996-aa37-dec314d8ff05%219720f782-289e-49a9-aa98-718cdf9dd3fd`

else:

`https://host.docker.internal:9200/external?contextRouteName=files-spaces-personal\u0026fileId=c3ce162e-6dd4-4abc-b402-d50d385a0573%247bd222b5-1d3d-4cb4-8de6-c84bf1f0aa30%210994538e-bece-4234-9acc-547b1417613b`

Reply:
then pattern can be
`"pattern": "%base_url%/external?contextRouteName=...`